addons/project/tests/test_project_flow.py

```diff
@@ -216,26 +216,26 @@ def test_rating(self):
 
         self.assertEqual(first_task.rating_count, 0, "Task should have no rating associated with it")
 
-        rating_good = self.env['rating.rating'].create({
+        rating_bad = self.env['rating.rating'].create({
```

Comment:
Is this change necessary ^^ ? Seems you are switching declaration but with same output ?

Reply:
Yep it is :) So that I do not have to change the whole test and just inverted the order of the non consumed and consumed.